ios/FluentUI/Tab Bar/SideTabBarTokens.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `TabBar`.
+open class SideTabBarTokens: ControlTokens {
+    /// The width of the SideTabBar
+    open var sideTabBarWidth: CGFloat { 62.0 }
+
+    /// The safe spacing for the avatar
+    open var avatarViewSafeTopSpacing: CGFloat { 18.0 }
+
+    /// The minimum spacing for the avatar
+    open var avatarViewMinTopSpacing: CGFloat { globalTokens.spacing[.xxLarge] }
+
+    /// The spacing for the avatar StackView
+    open var avatarViewTopStackViewSpacing: CGFloat { 34.0 }
+
+    /// The safe spacing for the bottom StackView
+    open var bottomStackViewSafeSpacing: CGFloat { 14.0 }
+
+    /// The min spacing for the bottom StackView
+    open var bottomStackViewMinSpacing: CGFloat { globalTokens.spacing[.xLarge] }
+
+    /// The spacing between the top set of TabBarItems
+    open var topTabBarItemSpacing: CGFloat { 32.0 }
+
+    /// The spacing between the bottom set of TabBarItems
+    open var bottomTabBarItemSpacing: CGFloat { globalTokens.spacing[.xLarge] }
+
+    /// The size of the top set of TabBarItems
+    open var topTabBarItemSize: CGFloat { globalTokens.iconSize[.large] }
+
+    /// The size of the bottom set of TabBarItems
+    open var bottomTabBarItemSize: CGFloat { globalTokens.iconSize[.medium] }
+
+    /// The padding for the badges on the top set of TabBarItems
+    open var badgeTopSectionPadding: CGFloat { 2.0 }
+
+    /// The padding for the badges on the bottom set of TabBarItems
+    open var badgeBottomSectionPadding: CGFloat { 4.0 }
```

Comment:
Seems like several of these paddings could be using a `globalTokens.spacing` value?